packages/common/src/api/account.ts

```diff
@@ -0,0 +1,28 @@
+import { createApi } from 'audius-query'
+
+type ResetPasswordArgs = {
+  email: string
+  password: string
+}
+
+const accountApi = createApi({
+  reducerPath: 'accountApi',
+  endpoints: {
+    resetPassword: {
+      async fetch(args: ResetPasswordArgs, context) {
+        const { email, password } = args
+        const { audiusBackend } = context
+
+        await audiusBackend.resetPassword(email, password)
+        return { status: 'ok' }
```

Comment:
do we need to check that the response code is good first?

Reply:
yeah good point, ill check what the saga does on desktop and bring this to parity

Reply:
ah so in this case, an "error" throws, which then results in the hook providing status === ERROR, so ill use that and update the ui.